src/ansys/fluent/core/fluent_connection.py

```diff
@@ -214,10 +214,10 @@ def __init__(
         )
 
         self.journal = Journal(self.scheme_eval)
+        self.version = self.scheme_eval.string_eval("(cx-version)")
```

Comment:
Let's process this right away. Essentially we need to apply ".".join(version.strip("()").split()). i.e. everything except the slice.